qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -0,0 +1,193 @@
+    fi
+    export ESHOME="/usr/share/elasticsearch"
+    export ESPLUGINS="$ESHOME/plugins"
+    export ESCONFIG="/etc/elasticsearch"
+    install() {
+        install_package
+        verify_package_installation
+    }
+fi
+
+@test "[$GROUP] install jvm-example plugin" {
+    install_jvm_example
+    remove_jvm_example
+}
+
+@test "[$GROUP] install jvm-example plugin with a custom path.plugins" {
+    # Clean up after the last time this test was run
+    rm -rf /tmp/plugins.*
+
+    local oldPlugins="$ESPLUGINS"
+    export ESPLUGINS=$(mktemp -d -t 'plugins.XXXX')
+
+    # Modify the path.plugins setting in configuration file
+    echo "path.plugins: $ESPLUGINS" >> "$ESCONFIG/elasticsearch.yml"
+    chown -R elasticsearch:elasticsearch "$ESPLUGINS"
+
+    install_jvm_example
+    remove_jvm_example
+}
+
+@test "[$GROUP] install jvm-example plugin with a custom CONFIG_DIR" {
+    # Clean up after the last time we ran this test
+    rm -rf /tmp/config.*
+
+    move_config
+
+    CONF_DIR="$ESCONFIG" install_jvm_example
+    CONF_DIR="$ESCONFIG" remove_jvm_example
+}
+
+@test "[$GROUP] install jvm-example plugin with a custom ES_JAVA_OPTS" {
+    # Clean up after the last time we ran this test
+    rm -rf /tmp/config.*
+
+    move_config
+
+    export ES_JAVA_OPTS="-Des.path.conf=$ESCONFIG"
+    install_jvm_example
+    remove_jvm_example
+}
+
+@test "[$GROUP] install jvm-example plugin from a directory with a space" {
+    rm -rf "/tmp/plugins with space"
+    mkdir -p "/tmp/plugins with space"
+    local zip=$(ls jvm-example-*.zip)
+    cp $zip "/tmp/plugins with space"
+
+    install_jvm_example "/tmp/plugins with space/$zip"
+    remove_jvm_example
+}
+
+@test "[$GROUP] install jvm-example plugin to elasticsearch directory with a space" {
+    [ "$GROUP" == "TAR PLUGINS" ] || skip "Test case only supported by TAR PLUGINS"
+
+    move_elasticsearch "/tmp/elastic search"
+
+    install_jvm_example
+    remove_jvm_example
+}
+
+@test "[$GROUP] install icu plugin" {
+    install_and_remove_special_plugin analysis icu icu4j-*.jar
+}
+
+@test "[$GROUP] install kuromoji plugin" {
+    install_and_remove_special_plugin analysis kuromoji
+}
+
+@test "[$GROUP] install phonetic plugin" {
+    install_and_remove_special_plugin analysis phonetic commons-codec-*.jar
+}
+
+@test "[$GROUP] install smartcn plugin" {
+    install_and_remove_special_plugin analysis smartcn
+}
+
+@test "[$GROUP] install stempel plugin" {
+    install_and_remove_special_plugin analysis stempel
+}
+
+@test "[$GROUP] install aws plugin" {
+    install_and_remove_special_plugin cloud aws aws-java-sdk-core-*.jar
+}
+
+@test "[$GROUP] install azure plugin" {
+    install_and_remove_special_plugin cloud azure azure-core-*.jar
+}
+
+@test "[$GROUP] install gce plugin" {
+    install_and_remove_special_plugin cloud gce google-api-client-*.jar
+}
+
+@test "[$GROUP] install delete by query" {
+    install_plugin delete-by-query "$(readlink -m delete-by-query-*.zip)"
```

Comment:
It didn't really have a "prefix". I bet I can get that working with the other. Let me see about that....